src/data/polynomial/monic.lean

```diff
@@ -179,6 +179,19 @@ begin
   rw [← hndeg, ← polynomial.leading_coeff, hp.leading_coeff, C.map_one]
 end
 
+lemma nat_degree_pow (hp : p.monic) (n : ℕ) :
+  (p ^ n).nat_degree = n * p.nat_degree :=
+begin
+  induction n with n hn,
+  { simp },
+  { rw [pow_succ, hp.nat_degree_mul (monic_pow hp n), hn],
+    ring }
+end
+
+lemma nat_degree_pow_X_add_C [nontrivial R] (n : ℕ) (r : R) :
```

Comment:
This doesn't need to be in the `monic` namespace, right? Also, it looks like a good simp-lemma to me.
```suggestion
@[simp] lemma nat_degree_pow_X_add_C [nontrivial R] (n : ℕ) (r : R) :
```